dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/PLL/PLL.mo

```diff
@@ -0,0 +1,125 @@
+within Dynawo.Electrical.Controls.PLL;
+
+model PLL "Phase-Locked Loop"
+
+  import Modelica.Blocks;
+  import Modelica.ComplexBlocks;
+  import Modelica.Constants;
+  import Modelica.SIunits;  
+    
+  import Dynawo.Connectors;
+
+  parameter SIunits.PerUnit Kp "PLL voltage calculation proportional gain";
```

Comment:
use Dynawo types: import Dynawo.Types and then Types.PerUnit (for all parameters)

Reply:
done